### ISSUE TITLE
Bug:

### ISSUE THREAD

**Issue by robboz4** [d0n4v4nb3ck3r/homebridge-honeywell-home#589]
### Describe The Bug

Temperature not accurately showing up on Homebridge

### To Reproduce

Normal operation

### Expected behavior

Correct Temperature dispalyed

### Relevant log output

```shell
[HoneywellHome] Initializing HoneywellHome platform...
[11/6/2021, 1:57:36 PM] [HoneywellHome] "Missing refreshToken"

[HoneywellHome] Loading accessory from cache: Master Bedroom Thermostat
[11/6/2021, 1:57:36 PM] [HoneywellHome] Loading accessory from cache: Hallway Thermostat
[11/6/2021, 1:57:36 PM] [HoneywellHome] Loading accessory from cache: Office IndoorAirSensor
```

### Config for homebridge-honeywell-home

```shell
"name": "HoneywellHome",
    "credentials": {
        "consumerKey": "--------------",
        "consumerSecret": "------------",
        "notice": "Keep your tokens a secret!"
    },
    "options": {
        "thermostat": {
            "thermostatSetpointStatus": "PermanentHold"
        },
        "roompriority": {
            "priorityType": "PickARoom"
        }
    },
    "platform": "HoneywellHome"
}
```

### Screenshots

<img width="165" alt="Screen Shot 2021-11-03 at 10 33 56 AM" src="https://user-images.githubusercontent.com/3642405/140623586-83b3477d-3544-44e3-8427-dd5168acca6a.png">

### Device & Model

T-10

### Node.js Version

v16.13.0

### NPM Version

v8.1.0

### Homebridge Version

v1.3.5

### Homebridge Honeywell Home Plugin Version

v11.0.2

### Homebridge Config UI X Plugin Version

v4.41.2

### Operating System

Linux Homebridge 5.10.60-v7+ #1449 SMP Wed Aug 25 15:00:01 BST 2021 armv7l GNU/Linux

**Comment by donavanbecker** [comment]
@robboz4 So guess what is the whole issue? I see the screenshots and then see missing refreshtoken which tells me that you need to relink your account.

**Comment by robboz4** [comment]
I couldn't relink due to this  error:
 {"code":400, "message":"The redirect URL provided does not match the redirect URL registered for the app."}
So I created a new APP and this time it linked and is reflecting the correct temperature. 

The first time I linked the APP it worked. So I guess it broke in an update. I've noticed that the thermostats tend to report occasional issues reporting tot he mothership. But the rest of my connected device are fine. So perhaps it was a Honeywell hiccup.

Thanks once again for the quick help.
